web_m2x_options/static/src/js/form.js

```diff
@@ -241,10 +241,20 @@ odoo.define('web_m2x_options.web_m2x_options', function (require) {
 
                 // create...
                 var create_edit = self.is_option_set(self.options.create) || self.is_option_set(self.options.create_edit),
-                    create_edit_undef = _.isUndefined(self.options.create) && _.isUndefined(self.options.create_edit),
+                    create_edit_undef = _.isUndefined(self.options.create) && _.isUndefined(self.options.create_edit) && _.isUndefined(self.options.no_create) && _.isUndefined(self.options.no_create_edit),
```

Comment:
So now we have `create_edit` and `no_create_edit` options? It seems a little bit absurd... IMHO, If Odoo core supports some of this addon's features, they should just be dropped in this addon, to DRY.

Reply:
Yes, I agree, it's absurd and they could be removed from this addons.
However, I'd suggest to keep them to make the module compatible with both attributes from the core, and attributes created before this commit.

But anyway, I'd refactor it completely in v11

Reply:
In such case, you should please add a note in Known Issues (in the README) so we don't forget dropping that in the migration.

Reply:
But you can put it here also as comment with `TODO`

Reply:
good idea :+1: